src/utils/embededPlayerMixin.js

```diff
@@ -0,0 +1,34 @@
+import { mapState } from 'vuex';
+import backendAuthMixin from './backendAuthMixin';
+
+export default () => ({
+  mixins: [backendAuthMixin()],
+  data() {
+    return {
+      isPlaying: false,
+      showCookiesDialog: false,
+    };
+  },
+  computed: {
+    ...mapState(['address']),
```

Comment:
The address is not used by this mixin. Import it explicitly in the components that depend on it.